### PR TITLE
1. added column isActive on listing.js,2. created a SQL dir with List…

### PR DESCRIPTION
**Pull request by Pnickolas1**
1. added column isActive, boolean value, defaulted to true on Listing.js
2. created a SQL_DummyData folder with Listings_Dummy and corresponding dummy data

### CODE REVIEW

**Review by a-ashbeck — COMMENTED** [review]
@Pnickolas1 You mentioned you made a change to the `listing.js` file? I am not seeing that here, just the dummy data addition.

**Comment by Pnickolas1** [comment]
@a-ashbeck sorry, I had to git add -A , you will see changes on Listings.js now

File: config/config.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "development": {
         "username": "root",
-        "password": null,
+        "password": "root",
+        "port": 8889,
```
Comment (a-ashbeck) [inline config/config.json]: @Pnickolas1 Looks like you forgot your login info in the `config.js`

File: models/listing.js [hunk 2]
```diff
@@ -19,6 +19,10 @@ module.exports = function(sequelize, DataTypes) {
                 notEmpty: true
             }
         },
+        isActive: {
+            type: DataTypes.BOOLEAN,
+            defaultValue: true
+        },
```
Comment (a-ashbeck) [inline models/listing.js:25]: Cool, this should work perfectly.

**Review by a-ashbeck — CHANGES_REQUESTED** [review]
This ought to be good to go once the login info is taken out. Otherwise, it looks good! Thanks!

**Comment by kshiro622** [comment]
We've merged pull requests since this once so I think you'll need to do a git fetch/rebase master.

**Review by a-ashbeck — APPROVED** [review]

**Pull request merged**